config/example/config.txt

```diff
@@ -314,4 +314,7 @@ PLAYER_REROUTE_CAP 0
 #DISABLE_LOBBY_MUSIC
 
 ## Uncomment this if you want to disable the popup alert for people on the same CID
-#DISABLE_CID_WARN_POPUP
\ No newline at end of file
+#DISABLE_CID_WARN_POPUP
+
+## Uncomment to use the iterative explosions (explosions that progress over time instead of instantly, less lag and near infinite explosion size support at lower lag)
+#use_iterative_explosions
```

Comment:
actually in the code its all in lowercase, but the code seem case insensitive so meh, i dont mind changing it in the example config.